docs/dao-operations/dao-roles.md

```
@@ -0,0 +1,200 @@
+---
+id: dao-roles
+title: DAO Roles empowered with Hats
+sidebar_label: DAO Roles (Hats)
+---
+
+In order to make operations more efficient and reduce the coordination needed at the share holder level, we've elected to create some discreet roles and give them budgets and other decision making authorities during their appointment.
+
+These roles are generally appointed by motivated individuals but elections can be facilitated if there are multiple Raiders interested in a role for the same term. These roles are created and appointed by the full community and can be removed/unwound in a similar fashion. We're continuing to use other good principal-agent hygiene by using quarterly budgets or streams in longer cases. Streams can be canceled with a proposal since the funds are flowing from the DAO itself.
+
+The roles here are powered by [Hats Protocol](https://app.hatsprotocol.xyz). Skip to [the RG Hats Tree section](#hats-tree) to learn more about [RaidGuild Hats Tree](https://app.hatsprotocol.xyz/trees/100/92) and [proposing a role](#proposing-a-role).
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)